src/common/contributor.ts

```diff
@@ -1,22 +1,27 @@
 import { Spec } from 'game/SPECS';
 import * as contributors from 'CONTRIBUTORS';
 
+import { HeartOfAzeroth } from './heartOfAzeroth';
+
 export type Character = {
```

Comment:
This is not the `character` you want to convert for use in the store or the api response. This is just the definition for contribs to declare their character links. You'll want to create a new type somewhere else. Changing the commas to semi's should stay though

Reply:
Done